src/App/global.js

```diff
@@ -0,0 +1,13 @@
+
+//This file is used to store all global variables
+
+
+//constant STUDENT_SESSION is used to enable/disable student applications to be filed for Student Sessions.
+//Value at index 0 is used to enable/disable the meny item which leads to the application screen.
+//Value at index 1 is used to enable/disable the submit button at the application screen.
+//Value at index 2 is used to display information appropriate for enabled/disabled student sessions at the student home screen.
+//Value at index 3 is used to display information appropriate for enabled/disabled student sessions at student session home screen.
+
+export const STUDENT_SESSION = [false, false, false, false];
+
+export default global;
```

Comment:
Ah, you don't have editorconfig enabled. https://editorconfig.org/

Our config (there should be a trailing newline at the end of the file): https://github.com/careerfairsystems/nexpo-web/blob/master/.editorconfig